src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -89,26 +98,36 @@ private static void implementCommand(List<String> parsedCommandList, List<String
                 }
                 Room roomKeyID = houseMap.get(roomExits.get(dirMovement.getDirection()));
                 player.move(roomKeyID);
-                break;
-            case "get":
-            case "grab":
+            }
+            case "get", "grab" -> {
                 String grabbedItem = UserInput.nounItemHelper(parsedCommandList, player);
                 player.addItem(grabbedItem);
                 playerCurrentRoom.removeItem(grabbedItem);
-                break;
-            case "drop":
+            }
+            case "drop" -> {
                 String droppedItem = UserInput.nounItemHelper(parsedCommandList, player);
                 player.dropItem(droppedItem);
                 playerCurrentRoom.addItem(droppedItem);
-                break;
-            case "talk":
-                // TODO: Player needs a Talk method to talk with characters in rooms
-//                    player.talkWith(output.get(1));
-//                    break;
-            case "inspect":
-                // TODO: Player needs an Inspect method to items in room
-//                    player.inspectItem(output.get(1));
-//                    break;
+            }
+            case "recruit" -> {
+                String recruitedNpc = UserInput.nounItemHelper(parsedCommandList, player);
+                player.addNpc(recruitedNpc);
+                playerCurrentRoom.removeNpc(recruitedNpc);
+                switch (recruitedNpc.toLowerCase()) {
```

Comment:
looks like there is no default statement for this. what happens if the user types in "recruit asdf"

Reply:
we can create a separate bug for this. looks like you satisfied the requirements for your ticket.